### ISSUE TITLE
Error [ERR_REQUIRE_ESM]

### ISSUE THREAD

**Issue by LightAndy1** [FalloutStudios/Reciple#35]
I get this error and don't know how to fix it (down below), please help me :)
```cmd
D:\Discord\DLEvolveX\node_modules\@reciple\client\dist\classes\errors\BaseError.js:10
const strip_ansi_1 = __importDefault(require("strip-ansi"));
                                     ^

Error [ERR_REQUIRE_ESM]: require() of ES Module D:\Discord\DLEvolveX\node_modules\strip-ansi\index.js from D:\Discord\DLEvolveX\node_modules\@reciple\client\dist\classes\errors\BaseError.js not supported.
Instead change the require of index.js in D:\Discord\DLEvolveX\node_modules\@reciple\client\dist\classes\errors\BaseError.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (D:\Discord\DLEvolveX\node_modules\@reciple\client\dist\classes\errors\BaseError.js:10:38)
    at Object.<anonymous> (D:\Discord\DLEvolveX\node_modules\@reciple\client\dist\index.js:23:14) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by NotGhex** [comment]
What version of reciple are you using?

**Comment by NotGhex** [comment]
This issue shoul've been fixed in 61229936c9abf396db1ab858056fd560146d685c
